YakimS/assignment1/Q1

```diff
@@ -0,0 +1,71 @@
+public class Assignment1Q1
+{
+
+    public static void main(String[] args)
+    {
+        String s1 = "si Apepl Apepl ppppo";
+        String s2 = "ppeAl ppopp Apepl is";
+        System.out.println(isSentenceAnagram(s1, s2,true));
+        System.out.println(s1 + "      " + s2);
+    }
+
+    static boolean isSentenceAnagram(String sentence1, String sentence2, boolean isCaseSensitive)
+    {
+        String[] sent1 = sentence1.split(" ");
+        String[] sent2 = sentence2.split(" ");
+
+        if(sent1.length != sent2.length) return false;
+
+        boolean[] isSent2WordAnagramFound = new boolean[sent2.length];
+
+        for(int i = 0 ; i < sent1.length ; i++)
+        {
+            for(int j = 0 ; j < sent2.length ; j++)
+            {
+                if (isSent2WordAnagramFound[j]) continue;
```

Comment:
Clever! It took me a while to understand what is the purpose of this line and that it is correct.

It would be good if you added some comments that would make it clear what is the algorithm supposed to do.